lib/cesar_cypher.ex

```diff
@@ -1,24 +1,29 @@
 defmodule CesarCypher do
   def main() do
     IO.puts "Give me your file name"
+    IO.read(:line) 
+    |> String.trim()
+    |> cesar_cypher()
   end
 
   def main(name) do
     read_cypher(name)
     |> cesar_cypher()
   end
 
-  @spec read_cypher(binary()) :: :ok | binary()
-  def read_cypher(name) when is_bitstring(name) do
+  @spec read_cypher!(binary()) :: binary()
+  def read_cypher!(name) when is_binary(name) do
```

Comment:
`is_binary/1` works for real strings. While `is_bitstring/1` would work for bit strings indivisible by 8 bits